### PR TITLE
Remove logo, banner, description and title from embedded groups

### PR DESCRIPTION
**Pull request by KeirOsborn**
## What? Why?

Part of task #1783

"Remove the logo, banner and Description text fields"

This pull enacts the above. I.e. It removed the whole header element from from the group page template if it's being viewed as an embedded page. This also includes the group name so that has also been removed (as instructed). 

## What should we test?
Tests in spec/features/consumer/shopping/embedded_groups_spec.rb

Also functionality can be manually tested using the embedded-group-preview.html file. i.e. by creating a group and then navigating to: http://localhost:3000/embedded-group-preview.html?your-group-name.

## Release notes
Third of four subtasks. Does not solve issue #1783 by itself.

### CODE REVIEW

**Comment by enricostano** [comment]
I'm starting to think that we probably need 2 separate views: default and embedded. Of course with shared behavior using partials etc. I just see too many "if embedded" in controller and view.

**Comment by KeirOsborn** [comment]
Ok well let me know because i'm working on the last part of this right now. 

The reason I broke it up like this is because it was required that a pull request was submitted for each part. I assumed that this was so that testers could play around with the functionality and cherry pick the parts they liked more easily. So that's how I did it. 

Obviously if this isn't a concern then I agree, it definitely makes more sense to have separate views.

**Comment by enricostano** [comment]
I don't think that the 2 things are mutually exclusive (small focused PRs and separate views), but I don't want to block anything now.

**Comment by KeirOsborn** [comment]
Well I'm very happy to proceed either way. Let me know what you think is best.

**Comment by lin-d-hop** [comment]
@KeirOsborn We dont care about testing parts separately. It was just to have small PRs that could be ticked off in small pieces of work.
If it's not going to cause the task to blow out do two separate views.

**Comment by KeirOsborn** [comment]
Ok cool sounds good will do.

**Comment by KeirOsborn** [comment]
Do you still want me to submit the subtasks as separate PRs though? Or it it ok if it's one big one? Given that all the work is now done.

**Comment by myriamboure** [comment]
Ideally #1783 should be split in multiple small stories, and each story should have a PR. Is it too late to do it for this one? If you have a good understanding on the stories done and remaining to be done, maybe it can be done? @lin-d-hop  what do you think? Would be great to avoid again the big PR ...

**Comment by lin-d-hop** [comment]
The PRs are small. Just the issue is 4 checkpoints rather than 4 issues. If you think it is the best use of your time @myriamboure then you could certainly create the small issues. But personally I think we just start doing that more diligently from now.

**Comment by myriamboure** [comment]
ok fine with me @lin-d-hop it was just a non-tech/process suggestion, but if it's small enough for you then just do what you think is best to finish this already-started case ;-)

**Comment by KeirOsborn** [comment]
Ok, so I think I'm just doing one PR with everything on it for this one. Is that right? In the grand scheme of things it still wont be a particularly big PR I don't think.

**Comment by daniellemoorhead** [comment]
@KeirOsborn let @mkllnk know when you want him to code review this (once you've got all the different bits into the one PR). Couldn't tell looking at this whether that had been done yet :)

File: spec/features/consumer/shopping/embedded_groups_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  Capybara.server_port = 9999
+
+  describe 'embedded groups' do
+	let(:enterprise) { create(:distributor_enterprise) }
+  	let!(:group) { create(:enterprise_group, enterprises: [enterprise], permalink: 'group1', on_front_page: true) }
+
+
+  	before do
+  	  Spree::Config[:enable_embedded_shopfronts] = true
+      Spree::Config[:embedded_shopfronts_whitelist] = 'localhost'
```
Comment (mkllnk) [inline spec/features/consumer/shopping/embedded_groups_spec.rb]: Just a minor thing: the whitespaces don't seem to be right. Tabs and spaces are mixed. Our convention is to use spaces only (no tabs) to preserve the same indentation no matter which tool you use.

**Comment by sauloperez** [comment]
Can we close this one then @KeirOsborn?

**Comment by KeirOsborn** [comment]
@sauloperez Yep, in fact this one, #2028 and #2029 can all be closed. I'll close them now.

**Comment by sauloperez** [comment]
Thanks!

**Comment by mkllnk** [comment]
@sigmundpetersen What was your trigger to re-open this pull request?

**Comment by sigmundpetersen** [comment]
Sorry, didn't mean to reopen. I was just trying to keep it together with the original issue for reference, but doesn't seem to work well in Zenhub. Closed again now.

**Pull request closed without merging**